src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -1477,7 +1469,20 @@ impl<'a, 'tcx> Visitor<'tcx> for Checker<'a, 'tcx> {
                     projection: None,
                 } = *place {
                     if NeedsDrop::in_local(self, local) {
-                        Some(self.body.local_decls[local].source_info.span)
+                        // Check whether the local hasn't been moved out of already.
+                        // FIXME(eddyb) use a more efficient dataflow cursor.
```

Comment:
Does this mean `DataflowResultsCursor` or are you thinking of something else? I guess there's no need to use a cursor now since we check the state at most once per block.

Reply:
Huh, why is `DataflowResultsCursor` not being used in the generator code, that's where I got this code from. I guess I can look into using the cursor API.